gobblin-data-management/src/test/resources/retentionIntegrationTest/testCombinePolicy/selection.conf

```diff
@@ -11,10 +11,10 @@ gobblin.retention : {
       combine.operation=INTERSECT
       combine.policy.classes=[
         gobblin.data.management.policy.SelectBeforeTimeBasedPolicy,
-        gobblin.data.management.policy.NonNewestKSelectionPolicy
+        gobblin.data.management.policy.NewestKSelectionPolicy
       ]
       timeBased.lookbackTime=3d
-      newestK.versionsSelected=2
+      newestK.versionsNotSelected=2
```

Comment:
We disabled the negative values for the options. The previous NonNewestKSelectionPolicy with newestK.versionsSelected=K  is now equivalent to NewestKSelectionPolicy with newestK.versionsNotSelected=K